docs/overview.yaml

```
@@ -414,11 +414,42 @@ Geometry:
   Differentiable manifolds:
     smooth or analytic manifold (with boundary and corners): 'IsManifold'
     smooth map between manifolds: 'ContMDiff'
+    product of manifolds: 'IsManifold.prod'
+    disjoint union of manifolds: 'IsManifold.disjointUnion'
+    interior of a manifold: 'ModelWithCorners.interior'
+    boundary of a manifold: 'ModelWithCorners.boundary'
     tangent bundle: 'TangentBundle'
     tangent map: 'tangentMap'
+    pullback of vector fields: VectorField.mpullback
+    Lie bracket of vector field: VectorField.mlieBracket
+    Jacobi/Leibniz identity: VectorField.leibniz_identity_mlieBracket
+    integral curves: 'IsIntegralCurve'
+    local existence of integral curves: exists_isIntegralCurveAt_of_contMDiffAt_boundaryless
+    uniform time existence: exists_isIntegralCurve_of_isIntegralCurveOn
```

I don't know this file well so this might be moot, but is it intentional that some of these entries are wrapped in quotation marks and some are not? All the other entries in this file are, so even if it doesn't matter it'd probably still make sense to maintain consistency.
```suggestion
    pullback of vector fields: 'VectorField.mpullback'
    Lie bracket of vector field: 'VectorField.mlieBracket'
    Jacobi/Leibniz identity: 'VectorField.leibniz_identity_mlieBracket'
    integral curves: 'IsIntegralCurve'
    local existence of integral curves: 'exists_isIntegralCurveAt_of_contMDiffAt_boundaryless'
    uniform time existence: 'exists_isIntegralCurve_of_isIntegralCurveOn'
```
The same applies to all the entries on fibre bundles too.

---

Good catch, thanks! It was not intentional; and I like the paranoia of "in yaml, always quote your strings". (Fun fact that this is sometimes really necessary: https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell) Changed.